ank/doc/swdesign/plantuml/seq_run_workload.plantuml

```diff
@@ -23,15 +23,6 @@ end box
 
 main -> cli_commands ++ : run workload
 
-cli_commands ->> listen_to_manager ++: request complete state
-listen_to_manager ->> server --++ : request complete state
-
-cli_commands -> cli_commands ++: wait for the response
-
-listen_to_server <<- server --++: complete state
-listen_to_server ->> cli_commands --: complete state
-cli_commands -> cli_commands ++: add workload
-
 cli_commands ->> listen_to_manager --++: request update state
```

Comment:
Same like above, I do not get why the communication is also done with the Agent Manager here..

Reply:
Same like above, I will discuss with @krucod3 and open a ticket for that.